src/OBJ/Assembler.elm

```diff
@@ -223,7 +219,22 @@ addFaceToMesh f mesh ({ vs, vts, vns, currentIndex } as state) =
 
         _ ->
             -- TODO: lift this error into a Result type
-            Debug.crash "mixed face types in the model!"
+            Debug.log "ERROR: mixed face types in the model!" <|
+
+            {      config = { withTangents=False }
```

Comment:
This isn't formatted by elm-format, right?
I assume you wanted to keep the diff clean (thanks for that), so you didn't run elm-format yet.
But I think it would also be fine if your last commit is actually just elm-format, since this also changed a bit since the 0.18 days.

Reply:
Sure, done!